src/tests/integration/omneo/profile/delete-profile-date.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, expect, test } from 'vitest'
+import { Omneo } from '../../../../omneo'
+import { ProfileDatesAttribute } from '../../../../types'
+import simpleOmneoRequest from '../../../lib/simple-omneo-request'
+import randomString from '../../../lib/string/random'
+
+const omneo = new Omneo({
+  tenant: process.env.OMNEO_TENANT as string,
+  token: process.env.OMNEO_TOKEN as string
+})
+
+const getHandle = () => { return `sdk_unit_test_dates_${randomString(5).toLowerCase()}_${Math.floor(Date.now() / 1000)}` }
+const testProfileID = process.env.OMNEO_TEST_PROFILE_ID as string
+
+describe('Profile Date Delete', () => {
+  test('SDK delete profile date', async () => {
+    const payload: ProfileDatesAttribute = {
+      name: 'Wedding',
```

Comment:
Could you please call this "SDK delete date"
In case one is left behind due to an error we can identify which test to fix

Reply:
can you please change the `name` attribute on the payload. 
Apologies was not more specific